spec/controllers/users_controller_spec.rb

```diff
@@ -17,23 +17,25 @@
   end
 
   describe 'POST #create' do
-    after { ActionMailer::Base.deliveries.clear }
-
     context "with valid input" do
-      before do
+      it 'should create the user' do
```

Comment:
Thanks! I'm going to try to be more precise in these descriptions as betterspecs.org recommends :+1: